src/components/Layout/Layout.js

```diff
@@ -49,7 +53,23 @@ const Layout = ({ children }) => {
         <PrimaryNav />
       </div>
       <main className="l-main" id="main-content">
-        {children}
+        <div data-test="main-children">{children}</div>
+        {!cookies.release20_04 && (
+          <Notification
+            type="information"
+            close={() => {
+              setCookies("release20_04", true);
+            }}
+          >
+            This is the new JAAS Dashboard, replacing the previous Juju GUI here
+            on jaas.ai and on your local Juju Controller from Juju 2.8.{" "}
+            <span className="u-hide--small">
+              Read more about this new product <a href="#_">on our blog</a> or{" "}
+              <a href="#_">discuss it on Discourse</a>. We would love to hear
```

Comment:
Can you create an issue to update these so we don't forget?

Reply:
They're less valuable than the first line and the alternative obscures the entire content area.